File: java/dicecup.java
import java.util.Scanner;

public class dicecup {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		int N = scan.nextInt();
		int M = scan.nextInt();
		int min = Math.min(N, M);
		for (int i = min + 1; i < N + M + 2 - min; i++)
			System.out.println(i);
		scan.close();
	}
}

File: ts/src/dicecup.ts
import readline = require('readline')

readline
  .createInterface({
    input: process.stdin,
    output: process.stdout
  })
  .on('line', (line: string) => {
    let [dice1, dice2] = line.split(' ').map(Number)
    dice1 ??= 0
    dice2 ??= 0

    const [min, max] = [Math.min(dice1, dice2), Math.max(dice1, dice2)]
    console.log(
      Array.from({ length: max - min + 1 }, (_, i) => min + i + 1).join('\n')
    )
  })
  .on('close', () => {
    process.exit(0)
  })
